deployments/helm-chart/README.md

```diff
@@ -184,6 +184,7 @@ Parameter | Description | Default
 `controller.enableCustomResources` | Enable the custom resources. | true
 `controller.enablePreviewPolicies` | Enable preview policies. | false
 `controller.enableTLSPassthrough` | Enable TLS Passthrough on port 443. Requires `controller.enableCustomResources`. | false
+`controller.enableCertManager` | Enable cert manager for Virtual Server resources. Requires controller.enableCustomResources. | false
```

Comment:
Is this enough information for someone to know why they'd enable this?

Perhaps a suggestion: `Enable x509 automated certificate management for VirtualServer resources using cert-manager (cert-manager.io).`